.circleci/config.yml

```diff
@@ -268,21 +252,10 @@ jobs:
 
   build-ios-beta:
     executor: ios
-    environment:
-      APP_NAME: "Mattermost Beta"
-      REPLACE_ASSETS: false
-      MAIN_APP_IDENTIFIER: "com.mattermost.rnbeta"
-      SENTRY_ENABLED: true
-      SEGMENT_API_KEY: ${SEGMENT_API_KEY_BETA}
-      MATCH_TYPE: appstore
-      MATCH_APP_IDENTIFIER: 'com.mattermost.rnbeta.NotificationService,com.mattermost.rnbeta.MattermostShare,com.mattermost.rnbeta'
-      EXTENSION_APP_IDENTIFIER: 'com.mattermost.rnbeta.MattermostShare'
-      NOTIFICATION_SERVICE_IDENTIFIER: 'com.mattermost.rnbeta.NotificationService'
-      IOS_APP_GROUP: 'group.com.mattermost.rnbeta'
-      IOS_ICLOUD_CONTAINER: 'iCloud.com.mattermost.rnbeta'
-      IOS_BUILD_EXPORT_METHOD: 'app-store'
-      AWS_FOLDER_NAME: 'mattermost-mobile-beta'
-      MATTERMOST_WEBHOOK_URL: ${MATTERMOST_WEBHOOK_BUILD}
+    environment: # remove env vars on final PR
+      SENTRY_ENABLED: false
+      AWS_FOLDER_NAME: ''
+      MATTERMOST_WEBHOOK_URL: ''
```

Comment:
threre was a value here before, is that intended?

Reply:
☝️